sdk/servicebus/azure-servicebus/tests/test_sb_client.py

```diff
@@ -76,6 +76,39 @@ def test_sb_client_entity_conflict(live_servicebus_config, standard_queue):
     with pytest.raises(AzureConflictHttpError):
         client.create_queue(standard_queue, lock_duration=300)
 
+@pytest.mark.liveTest
+def test_sb_client_queue_conflict_not_fail_on_exist(live_servicebus_config, standard_queue):
+
+    client = ServiceBusClient(
+        service_namespace=live_servicebus_config['hostname'],
+        shared_access_key_name=live_servicebus_config['key_name'],
+        shared_access_key_value=live_servicebus_config['access_key'],
+        debug=False)
+
+    client.create_queue(standard_queue, fail_on_exist=False)
+
+@pytest.mark.liveTest
+def test_sb_client_topic_conflict_not_fail_on_exist(live_servicebus_config, standard_topic):
+
+    client = ServiceBusClient(
+        service_namespace=live_servicebus_config['hostname'],
+        shared_access_key_name=live_servicebus_config['key_name'],
+        shared_access_key_value=live_servicebus_config['access_key'],
+        debug=False)
+
+    client.create_topic(standard_topic, fail_on_exist=False)
+
+@pytest.mark.liveTest
+def test_sb_client_subscription_conflict_not_fail_on_exist(live_servicebus_config, standard_subscription):
+
+    client = ServiceBusClient(
+        service_namespace=live_servicebus_config['hostname'],
+        shared_access_key_name=live_servicebus_config['key_name'],
+        shared_access_key_value=live_servicebus_config['access_key'],
+        debug=False)
+
+    client.create_subscription(standard_subscription, fail_on_exist=False)
```

Comment:
For completeness I'd probably validate the output (since fail_on_exist will silently return false if exists, true if not, rather than throwing exceptions), and might add one instance of an inverse test to validate failure.  (Especially since failure manifests slightly differently in the fail_on_exists true and false cases from within the management client.)

Reply:
I added several test conditions, and the test failed.

Can't see much info related, could you please have a look?

Reply:
Noticed that, have kicked them off to rerun and will keep an eye on that; some eng-sys pipeline changes went in today that I wouldn't _expect_ to impact this but the timing is suspicious.  Sorry about the interruption!  We'll get this sorted.